### ISSUE TITLE
Error installing Steam on Linux Mint

### ISSUE THREAD

**Issue by CiceroWesley** [ValveSoftware/steam-for-linux#7784]
#### Your system information

* Steam client version (build number or date):  1:1.0.0.70
* Distribution (e.g. Ubuntu):  Linux Mint 20.1
* Opted into Steam client beta?: [Yes/No] No 
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Hello, my Steam won't open. I have all the drives updated and the operating system as well. The error started to happen when I installed the new version of Linux Mint (20.1) before I used 19.3. Below I leave the photo of the error, I already tried to install the dependencies, however, I get the message that they are already installed. I also added the i386.

#### Steps for reproducing this issue:

1.  Installing Mint 20.1
2. Steam does not work

![error](https://user-images.githubusercontent.com/70028939/117158633-f9f13900-ad95-11eb-9b14-85dbe2c01129.png)

**Comment by kisak-valve** [comment]
Hello @CiceroWesley, if you run `steam` from a terminal, does the terminal spew give a hint? Also, please share the output of `apt policy libgl1-mesa-glx:i386`

**Comment by kisak-valve** [comment]
Thanks, let's ponder the output of `sudo apt install libc6:i386 libgl1:i386 libgl1-mesa-dri:i386`. If apt asks to remove a bunch of packages **do not** let the command proceed.

**Comment by CiceroWesley** [comment]
$ sudo apt install libc6: i386 libgl1: i386 libgl1-mesa-dri: i386
[sudo] password for wesley:
Reading package lists ... Done
Building dependency tree
Reading status information ... Ready
Some packages could not be installed. This may mean that
you requested an impossible situation or, if you are using a
unstable distribution, that some required packages have not been
created yet or have been removed from "Incoming".
The following information can help to resolve the situation:

The following packages have conflicting dependencies:
 common dictionaries: Depends on: debconf (> = 1.5.5) but will not be installed or
                                debconf-2.0
                       Depends: libtext-iconv-perl but will not be installed
E: Error, pkgProblemResolver :: Resolver generated failures, this can be packaged by collective packages (hold).

**Comment by kisak-valve** [comment]
`libtext-iconv-perl` is a weird dependency to get hung up on, and I don't think we've seen someone hit that packaging snafu on this issue tracker.

Unfortunately, that also means that I'm personally not going to be much help figuring out what's going on here, but someone else may have some ideas to try. There should be some folks in your distro's community who can also help try to figure out how to resolve this package conflict.

**Comment by kisak-valve** [comment]
I suspect there's some kind of auto-translation at work here and `dictionaries-common` is the actual name of the package apt is refusing to install.

**Comment by kisak-valve** [comment]
Can you check if that package is installed with something like `apt policy dictionaries-common` and complain the same way as 32 bit mesa if you try `sudo apt install dictionaries-common`?

**Comment by CiceroWesley** [comment]
Yes, there is a translation. I am Brazilian and I translate (Translator) into English.

apt policy dictionaries-common
dictionaries-common:
  Instalado: 1.28.1
  Candidato: 1.28.1
  Tabela de versão:
 *** 1.28.1 500
        500 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages
        500 http://archive.ubuntu.com/ubuntu focal/main i386 Packages
        100 /var/lib/dpkg/status


sudo apt install dictionaries-common
[sudo] senha para wesley:       
Lendo listas de pacotes... Pronto
Construindo árvore de dependências       
Lendo informação de estado... Pronto
dictionaries-common is already the newest version (1.28.1).
0 pacotes atualizados, 0 pacotes novos instalados, 0 a serem removidos e 6 não atualizados.

**Comment by CaptSilver** [comment]
Starting to sound like an issue with apt sources. You should double check your /etc/apt/sources.list.d directory to see what repos are installed. It should look like this: 

> deb http://packages.linuxmint.com ulyssa main upstream import backport
> 
> deb http://archive.ubuntu.com/ubuntu focal main restricted universe multiverse
> deb http://archive.ubuntu.com/ubuntu focal-updates main restricted universe multiverse
> deb http://archive.ubuntu.com/ubuntu focal-backports main restricted universe multiverse
> 
> deb http://security.ubuntu.com/ubuntu/ focal-security main restricted universe multiverse
> deb http://archive.canonical.com/ubuntu/ focal partner

After checking that run: 

> sudo apt --fix-broken install
> sudo dpkg --configure -a

If it asks to uninstall a large amount of packages, please cancel and submit a bug report to Linux Mint.

**Comment by CiceroWesley** [comment]
> Starting to sound like an issue with apt sources. You should double check your /etc/apt/sources.list.d directory to see what repos are installed. It should look like this:

Sources list

# Do not edit this file manually, use Software Sources instead.

deb http://packages.linuxmint.com ulyssa main upstream import backport #id:linuxmint_main

deb http://archive.ubuntu.com/ubuntu focal main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu focal-updates main restricted universe multiverse
deb http://archive.ubuntu.com/ubuntu focal-backports main restricted universe multiverse

deb http://security.ubuntu.com/ubuntu/ focal-security main restricted universe multiverse
deb http://archive.canonical.com/ubuntu/ focal partner

LANG=C sudo apt --fix-broken install 
[sudo] password for wesley:       
apt
Usage: apt command [options]
       apt help command [options]

Commands:
  add-repository   - Add entries to apt sources.list
  autoclean        - Erase old downloaded archive files
  autoremove       - Remove automatically all unused packages
  build            - Build binary or source packages from sources
  build-dep        - Configure build-dependencies for source packages
  changelog        - View a package's changelog
  check            - Verify that there are no broken dependencies
  clean            - Erase downloaded archive files
  contains         - List packages containing a file
  content          - List files contained in a package
  deb              - Install a .deb package
  depends          - Show raw dependency information for a package
  dist-upgrade     - Upgrade the system by removing/installing/upgrading packages
  download         - Download the .deb file for a package
  edit-sources     - Edit /etc/apt/sources.list with your preferred text editor
  dselect-upgrade  - Follow dselect selections
  full-upgrade     - Same as 'dist-upgrade'
  held             - List all held packages
  help             - Show help for a command
  hold             - Hold a package
  install          - Install/upgrade packages
  list             - List packages based on package names
  policy           - Show policy settings
  purge            - Remove packages and their configuration files
  recommends       - List missing recommended packages for a particular package
  rdepends         - Show reverse dependency information for a package
  reinstall        - Download and (possibly) reinstall a currently installed package
  remove           - Remove packages
  search           - Search for a package by name and/or expression
  show             - Display detailed information about a package
  showhold         - Same as 'held'
  showsrc          - Display all the source package records that match the given package name
  source           - Download source archives
  sources          - Same as 'edit-sources'
  unhold           - Unhold a package
  update           - Download lists of new/upgradable packages
  upgrade          - Perform a safe upgrade
  version          - Show the installed version of a package

LANG=C sudo dpkg --configure -a
